setup.py

```diff
@@ -55,8 +55,9 @@
     # installed, specify them here.  If using Python 2.6 or less, then these
     # have to be included in MANIFEST.in as well.
     package_data={
-        'allofplos':['seed.db'],
-        'allofplos.seed_corpus': ['*.xml'],
+        'allofplos': ['starter.db'],
+        'allofplos.starter_corpus': ['*.xml'],
```

Comment:
Is starter_corpus a python package? I don't know if this will always work correctly if it's not.  That doesn't need to be fixed for this PR, but it's worth considering for future PRs.